bucket/opensca-cli.json

```
@@ -0,0 +1,23 @@
+{
+    "version": "3.0.0",
+    "description": "OpenSCA-cli is a supply-chain security tool for security researchers and developers",
+    "homepage": "https://opensca.xmirror.cn",
+    "license": "Apache-2.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/XmirrorSecurity/OpenSCA-cli/releases/download/v3.0.0/opensca-cli-v3.0.0-windows-amd64.zip",
+            "hash": "md5:dc37f1c2138563516774ab02d201cfdf"
+        }
+    },
+    "bin": "opensca-cli.exe",
+    "checkver": {
+        "github": "https://github.com/XmirrorSecurity/OpenSCA-cli"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/XmirrorSecurity/OpenSCA-cli/releases/download/v$version/opensca-cli-v$version-windows-amd64.zip"
+            }
+        }
```

```suggestion
        },
        "hash": {
            "url": "$url.md5"
        }
```

Add link to hash file(s) as well.